R/fortify.anosim.R

```diff
@@ -0,0 +1,49 @@
+##' @title Fortify analysis of similarities (ANOSIM) results
+##' @description Prepares fortified versions of results from \code{\link[vegan]{anosim}} objects.
+##' @details TODO
```

Comment:
I'd remove `@details` unless you want to add something that doesn't duplicate what's in `@description`? It isn't necessary for the Rd. I may have included this in my roxygen tags but that is probably not good behaviour is more people are going to be contributing to the development of the package.

Reply:
Tag removed.